src/Microsoft.TemplateEngine.Utils/PhysicalFileSystem.cs

```diff
@@ -44,6 +44,10 @@ public void FileCopy(string source, string target, bool overwrite)
 
         public void DirectoryDelete(string path, bool recursive)
         {
+            foreach(string file in Directory.GetFiles(path, "*", SearchOption.AllDirectories))
```

Comment:
Okay.  I only added this because it wasn't letting me delete my scratch folder with the cloned git repo.

I will find another solution.